torch/hub.py

```diff
@@ -752,3 +761,17 @@ def load_state_dict_from_url(
     if _is_legacy_zip_format(cached_file):
         return _legacy_zip_load(cached_file, model_dir, map_location, weights_only)
     return torch.load(cached_file, map_location=map_location, weights_only=weights_only)
+
+
+def set_proxy(protocol, host, port):
```

Comment:
Could you add some docstrings here since it's a public method?
Would be nice to have a two line example of how to use it.

Reply:
Added docstrings for the public methods. The example is short enough to fit in one line.